src/main/java/serenitylabs/tutorials/vetclinic/collections/katas/questions/TheBookingStatus.java

```diff
@@ -0,0 +1,41 @@
+package serenitylabs.tutorials.vetclinic.collections.katas.questions;
+
+import net.serenitybdd.screenplay.Actor;
+import net.serenitybdd.screenplay.Question;
+import serenitylabs.tutorials.vetclinic.Pet;
+import serenitylabs.tutorials.vetclinic.collections.katas.BookingResponse;
+
+/**
+ * Created by vdheer on 9/20/2016.
+ */
+public class TheBookingStatus implements Question<Boolean> {
+
+    private BookingResponse bookingResponse = new BookingResponse(true);
+
+    private Pet pet;
+
+    public TheBookingStatus(Pet pet) {
+        this.pet = pet;
```

Comment:
The pet is never actually used. Remember, the booking status isn't directly related to the pet, but is the outcome the booking process when you try to book a pet into a hotel.